src/tests/ftest/container/simple_create_delete_test.py

```diff
@@ -20,67 +20,63 @@
   provided in Contract No. B609815.
   Any reproduction of computer software, computer software documentation, or
   portions thereof marked with this legend must also reproduce the markings.
-'''
-from __future__ import print_function
-
-import time
-import traceback
-from avocado import main
+"""
 from apricot import TestWithServers
-
-from pydaos.raw import DaosApiError, c_uuid_to_str
-from general_utils import get_pool, get_container
+from test_utils import TestPool, TestContainer
```

Comment:
(pylint-no-name-in-module) No name 'TestPool' in module 'test_utils'
(pylint-no-name-in-module) No name 'TestContainer' in module 'test_utils'